### ISSUE TITLE
OBF mail list issues

### ISSUE THREAD

**Issue by cjfields** [biojava/biojava#717]
(To the Biojava developers : this is per the OBF Board Meeting, feel free to edit/add to the ticket or text as needed)

As @peterjc has noted on the [main OBF blog](https://news.open-bio.org/2017/12/15/mailing-list-consolidation/), the OBF hosted mail lists (including the various Biojava lists) have had serious issues over the last several months due to heavy spam traffic on the OBF mailman server. We are soliciting community feedback from both Biojava developers and users for the next steps forward on resolving this issue (see below). Please post suggestions or concerns here.

We (OBF) are migrating to a hosted solution that should provide more stability, with the downside that many lists will be shut down and archived due to hosting costs. The blog post details some suggestions.

Again, please note your concerns as soon as possible. We would like feedback by Dec. 28 so that we can decide on the next steps.

Thanks all!

@cjfields

**Comment by josemduarte** [comment]
Thanks @cjfields ! As discussed with @peterjc already, in my opinion we are ok to keep only the biojava-l maling list, dropping biojava-dev. Github issues/pull requests are a really good solution for developers to communicate.

**Comment by sbliven** [comment]
I agree with @josemduarte on this. I'd be tempted to drop biojava-l as well since it doesn't get too much traffic, but I do want to have some way for people to get in touch that doesn't require a github account.

Let's officially deprecate biojava-dev and remove it from the website.

**Comment by peterjc** [comment]
If you need a hand making sure all the biojava-dev subscribers are on biojava-l, or updating the mailman description let me know. I did this for Biopython this week http://mailman.open-bio.org/pipermail/biopython/2018-January/016339.html

**Comment by sbliven** [comment]
@peterjc Your help would be very appreciated. If you could do the same for biojava-dev that would be great!

**Comment by peterjc** [comment]
OK, to move the ``biojava-dev`` subscribers over to ``biojava-l``, I'll be using ``mailman-subscribers.py`` (dated 2015-12-04) as per https://wiki.list.org/DOC/How%20do%20I%20extract%20%28export%29%20a%20list%20of%20my%20list%27s%20members%20%28subscribers%29%3F

This downloads the email addresses only, which makes it trivial to do the overlap computation:

```
$ python mailman-subscribers.py -o biojava-dev.txt mailman.open-bio.org biojava-dev XXX
$ wc -l biojava-dev.txt 
     284 biojava-dev.txt
$ python mailman-subscribers.py -o biojava-l.txt mailman.open-bio.org biojava-l XXX
$ wc -l biojava-l.txt 
     503 biojava-l.txt
$ python -c "l = set(open('biojava-l.txt')); d = set(open('biojava-dev.txt')); print(len(l.intersection(d)))"
134

```

Adding ``-f`` downloads the subscribers with their declared names, which is more useful for then mass-subscribing people. As of now, the combined list has 653 subscribers.

I've updated the description for ``biojava-dev`` and for the short term set everyone's moderation bit to block posting to the list (suggested by @hlapp). For ``biopython-dev`` I went further and unsubscribed everyone. Shall I do that here too?

I'm about to email the BioJava lists.
